x/nexus/keeper/chain.go

```diff
@@ -125,6 +138,16 @@ func (k Keeper) RemoveChainMaintainer(ctx sdk.Context, chain exported.Chain, mai
 	return nil
 }
 
+// GetMinAmount returns the asset's minimum transferable amount for the given chain
+func (k Keeper) GetMinAmount(ctx sdk.Context, chain exported.Chain, asset string) sdk.Int {
```

Comment:
Do you not want to return `(sdk.Int, bool)` so that the caller can know when the asset isn't even registered for the given chain?